test/integration/scheduler/priorities_test.go

```diff
@@ -245,10 +242,8 @@ func makeContainersWithImages(images []string) []v1.Container {
 	return containers
 }
 
-// TestEvenPodsSpreadPriority verifies that EvenPodsSpread priority functions well.
-func TestEvenPodsSpreadPriority(t *testing.T) {
-	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.EvenPodsSpread, true)()
-
+// TestPodTopologySpreadScore verifies the PodTopologySpread score plugin functions well.
+func TestPodTopologySpreadScore(t *testing.T) {
```

Comment:
avoid rename

Reply:
Change reverted. I will use another cleanup PR to cover this.